docs/migrate.rst

```diff
@@ -9,6 +9,18 @@ Minimum Python version is 3.8.0
 
 Rally 1.5.0 requires Python 3.8.0. Check the :ref:`updated installation instructions <install_python>` for more details.
 
+Meta-Data for queries are omitted
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Rally 1.5.0 does not determine query meta-data anymore by default to reduce the risk of client-side bottlenecks. The following meta-data fields are affected:
+
+* ``hits``
+* ``hits_relation``
+* ``timed_out``
+* ``took``
+
+If you still want to retrieve them (risking skewed results due to additional overhead), set the new property ``detailed-results`` to ``true`` for any operation of type ``search``.
```

Comment:
This contradicts what we are saying in https://github.com/elastic/rally/pull/935/commits/4fcaf9d0a17ffce744d424777c3cbc20d19caa8a#diff-a4e117fd9659940c6f1756d76f5dfe5cR405, i.e. that at least for the `search` operation the default is `true`?

Reply:
I changed this fo `false` already in https://github.com/elastic/rally/pull/935/commits/4c2e78a125a6b98fff25010c16066275e61f53d2?

Reply:
Ah! Ok then.